README.md

```diff
@@ -41,12 +41,14 @@ To generate a Rails application using this template, pass the `-m` option to `ra
 rails new blog \
   -d postgresql \
   -m https://raw.githubusercontent.com/Wolfpack-Digital/rails-template/master/template.rb
+  -j webpack
```

Comment:
You're updating the documentation with the `-j webpack` option but you also removed webpacker gem. This also causes the template to fail and not complete successfully
![Screenshot 2022-03-24 at 13 35 32](https://user-images.githubusercontent.com/9211589/159909105-29f9637f-770f-4432-b11a-8024ecf4af4e.png)
.

Reply:
It was missing some gems which are required for the installation. Somehow for me was working with every version, when testing it locally. Now I added the missing gems and retested it. Would be helpful if somebody can test it too on his PC.